src/ZlibSharp/ZlibSharp/MemoryZlib.cs

```diff
@@ -281,16 +118,23 @@ public static string ZlibVersion()
     /// <summary>
     /// Gets the Adler32 checksum of the input data at the specified index and length.
     /// </summary>
-    /// <param name="data">The data to checksum.</param>
+    /// <param name="Data">The data to checksum.</param>
     /// <returns>The Adler32 hash of the input data.</returns>
-    public static unsafe ulong ZlibGetAdler32(byte[] data)
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    public static ulong ZlibGetAdler32(byte[] Data)
+    {
+        return ZlibGetAdler32(Data.AsSpan());
+    }
+    

```

Comment:
```suggestion
```

Cant we remove the array version so that way the implicit conversion to ReadOnlySpan can be used for the Span version of this?